core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -78,7 +78,7 @@
         <source>E-mail address of the website administrator</source>
       </trans-unit>
       <trans-unit id="tl_page.adminEmail.1">
-        <source>Auto-generated messages like subscription confirmation e-mails will be sent to this address.</source>
+        <source>This e-mail address will be used to send system messages.</source>
```

Comment:
Shouldn't it say both?

Reply:
You mean "This e-mail address will be used to send and receive system messages"?

Reply:
Yeah, I think that would be best. Just to quickly recap what it is used for:

* Default sender address for any emails sent by Contao or its core extensions.
* Recipient address for notifications of new comments (unless specified otherwise).
* Recipient address for notifications of new member registrations (if enabled).

And that's about it I think?

Reply:
Yes.